src/pages/Detail/Detail.js

```diff
@@ -1,9 +1,149 @@
 import React from 'react';
 
+import StoreImgBox from './StoreImgBox/StoreImgBox';
+import StoreInfo from './StoreInfo/StoreInfo';
+import StoreReviewBox from './StoreReviewBox/StoreReviewBox';
+
+import './Detail.scss';
+
 class Detail extends React.Component {
+  state = {
+    restaurants: null,
+    food: null,
+    image: null,
+    review: [],
+  };
+
+  reviewRequestNum = 1;
+  reviewRequestNumLimit; //한 번에 10개씩 띄울 경우 총 리뷰 페이지 수
+
+  restaurantsAddr = `restaurants/${this.props.match.params.id}`;
+  foodsAddr = `restaurants/${this.props.match.params.id}/food`;
+  imgAddr = `restaurants/${this.props.match.params.id}/food/image`;
+  reviewsAddr = `restaurants/${this.props.match.params.id}/review?limit=${this.reviewRequestNum}`;
+
+  componentDidMount = () => {
+    this.fetchData(this.restaurantsAddr);
+    this.fetchData(this.foodsAddr);
+    this.fetchData(this.imgAddr);
+    this.fetchReviewData(1, 5);
+  };
+
+  setRequestNumLimit = () => {
+    const { restaurants } = this.state;
+
+    restaurants.review_count.total % 10 === 0
+      ? (this.reviewRequestNumLimit = restaurants.review_count.total / 10)
+      : (this.reviewRequestNumLimit =
+          Math.floor(restaurants.review_count.total / 10) + 1);
+  };
+
+  handleReviewEdit = () => {
+    console.log('edit');
+  };
+
+  handleReviewDel = reviewId => {
+    fetch(
+      `http://${IP_ADDRESS}:8000/restaurants/${this.props.match.params.id}/review`,
```

Comment:
```suggestion
    fetch(`${IP_ADDRESS}/restaurants/${this.props.match.params.id}/review`,
```
이런 식으로 프로토콜과 포트도 같이 묶어주면 좋을거 같아요